src/components/BalanceInfo/index.tsx

```diff
@@ -14,8 +14,9 @@ import { ReceiveTransaction } from "components/ReceiveTransaction";
 import { LayoutSection } from "components/LayoutSection";
 import { NATIVE_ASSET_CODE } from "constants/settings";
 import { AppDispatch } from "config/store";
-import { startAccountWatcherAction } from "ducks/account";
+// import { startAccountWatcherAction } from "ducks/account";
```

Comment:
The new TS Wallet SDK doesn't support account and history watchers yet, so I'm commenting this code out to make it easier to update it later.